lib/EnsEMBL/REST/Controller/Info.pm

```diff
@@ -331,6 +331,131 @@ sub consequence_types_GET {
   return;
 }
 
+### The following methods were ported from EG REST...
+
+sub ensgen_version : Chained('/') PathPart('info/eg_version') :
+  ActionClass('REST') : Args(0) { }
+
+sub ensgen_version_GET {
+  my ( $self, $c ) = @_;
+  $c->log()->info("Retrieving EG version from registry");
+  # lazy load the registry
+  $c->model('Registry')->_registry();
+  my $lookup = $c->model('Registry')->_lookup();
+  $self->status_ok( $c, entity => { version => $lookup->adaptor->data_release->ensembl_genomes_version } );
+  return;
+}
+
+sub genomes_name : Chained('/') PathPart('info/genomes') :
+  ActionClass('REST') : Args(1) { }
+
+sub genomes_name_GET {
+  my ( $self, $c, $name ) = @_;
+  # lazy load the registry
+  $c->model('Registry')->_registry();
+  my $lookup = $c->model('Registry')->_lookup();
+  my $expand = $c->request->param('expand');
+  $c->log()->info("Retrieving information about all genomes");
+  my $info = $lookup->adaptor()->fetch_by_any_name($name);
+  $c->go( 'ReturnError', 'custom', ["Genome $name not found"] )
+  unless defined $info;
```

Comment:
If you're determined to use the postfix, indent this please

Reply:
Changed formatting